Doc/library/functions.rst

```diff
@@ -1561,7 +1561,9 @@ are always available.  They are listed here in alphabetical order.
    returns ``100``, but ``pow(10, -2)`` returns ``0.01``.  For a negative base of
    type :class:`int` or :class:`float` and a non-integral exponent, a complex
    result is delivered.  For example, ``pow(-9, 0.5)`` returns a value close
-   to ``3j``.
+   to ``3j``. Whereas, for a negative base of type :class:`int` or :class:`float`
+   with an integral exponent, a real number is delivered. For example, 
```

Comment:
nitpick-level, but could we say "float result" instead of "real number", for consistency with the earlier part of this paragraph?

Hmm, but then this isn't quite accurate, since `pow(-9, 2)` returns an `int`, not a `float`. I suppose that one could argue that "real number" covers both cases, but that seems a bit of a stretch.

Reply:
I guess the all-operands `int` case was already covered earlier in the paragraph, though, so "float result" is probably okay here.

Reply:
Thanks for reviewing this, highly appreciated !